Modify the inaccuracies in the TLS description (#15312)
pingcap/docs: tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md

3. On the **Standard Connection** tab of this dialog, follow the three steps to set up the TLS connection.
   - Step 1：Create traffic filter

>    - Step 2：Download CA Cert

Reviewer: ```suggestion
   - Step 2：Download CA cert
```

   - Step 3：Connect with an SQL client

4. Under **Step 1: Create traffic filter** in the dialog, configure the IP addresses that are allowed to access your cluster. For more information, see [Configure an IP access list in standard connection](/tidb-cloud/configure-ip-access-list.md#configure-an-ip-access-list-in-standard-connection).

> 5. Under **Step 2: Download CA Cert**, click **Download CA Cert** to download it locally for client TLS configuration. The CA Cert ensures that the TLS connection is secure and reliable.

Reviewer 2: ```suggestion
5. Under **Step 2: Download CA Cert**, click **Download CA Cert** to download it locally for client TLS configuration. The CA certificate ensures that the TLS connection is secure and reliable.
```

Author: CA certificate and CA Cert both mean the same thing. Our competing product is also described as CA Cert. There is no need to modify it here.

> 5. Under **Step 2: Download CA Cert**, click **Download CA Cert** to download it locally for client TLS configuration. The CA Cert ensures that the TLS connection is secure and reliable.

Reviewer: ```suggestion
5. Under **Step 2: Download CA cert**, click **Download CA cert** to download it locally for client TLS configuration. The CA cert ensures that the TLS connection is secure and reliable.
```

    > **Note:**
    >

>     > - You can store the downloaded CA Cert in the default storage path of your operating system, or specify another storage path. You need to replace the CA Cert path in the code example with your own CA Cert path in the subsequent steps.  

Reviewer: ```suggestion
    > - You can store the downloaded CA cert in the default storage path of your operating system, or specify another storage path. You need to replace the CA cert path in the code example with your own CA cert path in the subsequent steps.  
```

>     > - After downloading your CA Cert, you can store it in the default storage path of your operating system, or specify another storage path. You need to replace the CA Cert path in the code example with your own CA Cert path in the subsequent steps.  

Reviewer 3: ```suggestion
    > - You can store the downloaded CA Cert in the default storage path of your operating system, or specify another storage path. You need to replace the CA Cert path in the code example with your own CA Cert path in the subsequent steps.  
```

>     > After downloading your CA Cert, you can store it in the default storage path of your operating system, or specify another storage path. You need to replace the CA Cert path in the code example with your own CA Cert path in the subsequent steps.  
>    > TiDB Dedicated does not force clients to use TLS connections, and user-defined configuration of the `require_secure_transport` variable is not supported on [TiDB Dedicated](https://docs.pingcap.com/tidbcloud/system-variables#require_secure_transport-new-in-v610) currently.

Reviewer: ```suggestion
    > - After downloading your CA Cert, you can store it in the default storage path of your operating system, or specify another storage path. You need to replace the CA Cert path in the code example with your own CA Cert path in the subsequent steps.  
   > - TiDB Dedicated does not force clients to use TLS connections, and user-defined configuration of the  [`require_secure_transport`](/system-variables.md#require_secure_transport-new-in-v610) variable is currently not supported on [TiDB Dedicated](https://docs.pingcap.com/tidbcloud/system-variables#require_secure_transport-new-in-v610).
```

>    > - TiDB Dedicated does not force clients to use TLS connections, and user-defined configuration of the [`require_secure_transport`](/system-variables.md#require_secure_transport-new-in-v610) variable is currently not supported on [TiDB Dedicated](https://docs.pingcap.com/tidbcloud/system-variables#require_secure_transport-new-in-v610).

Reviewer: ```suggestion
   > - TiDB Dedicated does not force clients to use TLS connections, and user-defined configuration of the [`require_secure_transport`](/system-variables.md#require_secure_transport-new-in-v610) variable is currently not supported on TiDB Dedicated.
```

No.

> TiDB Dedicated only supports one-way TLS authentication, and does not support two-way TLS authentication currently. Please do not make custom configurations of `ssl_cert` and `ssl_key` parameters on TiDB Dedicated. If you need two-way TLS authentication, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).

Reviewer: ```suggestion
TiDB Dedicated only supports one-way TLS authentication, and does not support two-way TLS authentication currently. If you need two-way TLS authentication, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
```

Reviewer: reason: these two variables are read-only on TiDB Cloud.
